packages/common/src/services/integrations/cleeng/CleengAccountService.ts

```diff
@@ -67,16 +76,29 @@ export default class CleengAccountService extends AccountService {
     return decodedToken.customerId;
   };
 
-  private getCustomer: GetCustomer = async (payload, sandbox) => {
-    return this.cleengService.get(sandbox, `/customers/${payload.customerId}`, { authenticate: true });
+  private getCustomer: GetCustomer = async (payload) => {
+    return this.cleengService.get(`/customers/${payload.customerId}`, { authenticate: true });
   };
 
-  private getLocales: GetLocales = async (sandbox) => {
-    return this.cleengService.getLocales(sandbox);
+  private getLocales: GetLocales = async () => {
+    return this.cleengService.getLocales();
   };
 
-  initialize = async (config: Config, logoutCallback: () => Promise<void>) => {
-    await this.cleengService.initialize(!!config.integrations.cleeng?.useSandbox, logoutCallback);
+  initialize = async (config: Config, _url: string, logoutCallback: () => Promise<void>) => {
+    const cleengConfig = config?.integrations?.cleeng;
```

Comment:
Should we add a Config dependency?

Reply:
What do you mean exactly?

Reply:
Never mind, was thinking about store being accessible as a dependency.